wdl-lint/RULES.md

```diff
@@ -7,7 +7,7 @@ be out of sync with released packages.
 ## Lint Rules
 
 | Name                             | Tags                          | Description                                                                           |
-|:---------------------------------|:------------------------------|:--------------------------------------------------------------------------------------|
+| :------------------------------- | :---------------------------- | :------------------------------------------------------------------------------------ |
```

Comment:
This is due to the Markdown All-in-One extension formatting from within VSCode. I suppose it's fine unless anyone has any issues—that makes it much easier to keep the spacing consistent (for me at least).